### DISCUSSION TITLE
How to create fine grained access tokens for selected repositories that already exist

### DISCUSSION THREAD

**Discussion by timethernom** [community/community#131599] [Repositories] [answered]
### Select Topic Area

Question

### Body

When I first tried to create a fine-grained access token, I found that none of my repositories were visible under "Selected repositories".  After I enrolled my Organization in the fined-grained access token beta program, I found that I could create an access token for a newly created repository, but not for my existing repositories.

Is there are way to retroactively enable my existing repositories for fine-grained access control?  Could I leave the Organization, or delete it?  I'm not sure what function the Organization really plays, or when/why I created it.  I just have my own account in it.

**Comment by timethernom** [comment, marked as answer]
The solution was to click on Resource Owner pop-up and choose my organization as the owner.  Then, all of my repos were visible for selection.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
